L3.ipynb

```diff
@@ -77,27 +114,48 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {
-    "collapsed": false
+    "collapsed": false,
+    "slideshow": {
+     "slide_type": "fragment"
+    }
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Prosze nie umieszczac notatnikow z jakimkolwiek "wyjsciem". Przed umieszczeniem czegokolwiek w repozytorium prosze wyczyscic wszystkie komorki wyjsciowe. Tyczy sie to wszystkich plikow